package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "types": "index.d.ts",
   "scripts": {
-    "docs": "./docs/scripts/build_docs.sh apm-agent-nodejs ./docs ./build",
```

Comment:
We have a Travis test that currently expects this script to be there:

https://github.com/elastic/apm-agent-nodejs/blob/9abb4aa926fdca65e70c570735f8f91985905952/.travis.yml#L63-L67

So we should either update that or remove the build from Travis, which would also fix #1071

Reply:
Ah, thanks, forgot we talked about this as well!